Helper/Entity/Product/PriceManager.php

```diff
@@ -139,7 +139,7 @@ public function addPriceData($customData, Product $product, $subProducts)
                         );
                     }
 
-                    if ($customData[$field][$currencyCode]['default'] === 0) {
+                    if ($customData[$field][$currencyCode]['default'] == 0) {
```

Comment:
I'd rather not drop the strict comparison, but have it compared with `0.0` float value.

Reply:
Why? There is no harm in doing a loose comparison here. A falsey value should always hit the zerodefault method, no matter the reason why.

Reply:
I'd even go as far as changing the line to

`if (!$customData[$field][$currencyCode]['default']) {`

Since, as I said, *any* falsey value should hit the handleZeroDefaultPrice method.

Reply:
Makes sense, can you change the `== 0` to `!` statement?

Reply:
Done :+1: